assets/js/blocks/cart-checkout/checkout/frontend.js

```diff
@@ -10,6 +11,16 @@ import Block from './block.js';
 import blockAttributes from './attributes';
 import renderFrontend from '../../../utils/render-frontend.js';
 
+/**
+ * Wrapper component to supply API data.
+ *
+ * @param {Object} attributes object of key value attributes passed to block.
+ */
+const CheckoutFrontend = ( attributes ) => {
+	const { shippingRates } = useStoreCart();
+	return <Block attributes={ attributes } shippingRates={ shippingRates } />;
+};
```

Comment:
We'll leave this for now, but I still think it will be better to handle this via an `isEditor` flag that is passed through via block registration of the `Editor` component loading the checkout block because there will be more contexts we need to know the editor context. 

Reason why we'll leave it for a separate pull is this will also be affected by the checkout flow changes I'll be proposing.

Reply:
all hooks that loads external data on load will need this flag (load from store or get from preview), this can get tedious and messy in our code by time (mixing different logic in a single production hook or component), somehow along the way we need a more robust solution.